.circleci/config.yml

```diff
@@ -5,8 +5,10 @@
 #  Distributed under the Boost Software License, Version 1.0. (See accompanying
 #  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-defaults: &defaults
+working_dir_default: &working_dir_default
```

Comment:
Isn't it better to keep the target `defaults` depending on `working_dir_default` and `docker_default`?
```
defaults: &defaults
    <<: *docker_default
    <<: *working_dir_default
```